sdk/communication/Azure.Communication.NetworkTraversal/src/Models/GetRelayConfigurationOptions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.NetworkTraversal.Models
+{
+    /// <summary> GetRelayConfigurationOptions. </summary>
+    public class GetRelayConfigurationOptions
+    {
+        /// <summary> The <see cref="CommunicationUserIdentifier"/> for whom to issue a token. </summary>
+        public CommunicationUserIdentifier CommunicationUser { get; set; }
+
+        /// <summary> The specified <see cref="RouteType"/> for the relay request. </summary>
+        public RouteType? RouteType { get; set; }
+
+        /// <summary> Initializes a new instance of GetRelayConfigurationOptions. </summary>
+        public GetRelayConfigurationOptions()
+        {
+        }
+
+        /// <summary> Initializes a new instance of GetRelayConfigurationOptions. </summary>
+        /// <param name="communicationUser"> The <see cref="CommunicationUserIdentifier"/> for whom to issue a token. </param>
+        /// <param name="routeType"> The specified <see cref="RouteType"/> for the relay request. </param>
+        public GetRelayConfigurationOptions(CommunicationUserIdentifier communicationUser = null, RouteType? routeType = null) {
```

Comment:
For options we don't create constructors, instead developers will use object initialization style.
So let's delete the constructor here, and developers will write
```c#
new GetRelayConfigurationOptions { CommunicationUser = user, RouteType = RouteType.Nearest }
```